src/main/java/name/valery1707/junit/rule/ConditionalIgnoreRule.java

```diff
@@ -92,11 +109,22 @@ public void evaluate() {
 
 	@Retention(RetentionPolicy.RUNTIME)
 	@Target({ElementType.METHOD})
+	@Repeatable(ConditionalIgnoreList.class)
```

Comment:
\[Checkstyle\] WARNING: 'annotation def modifier' has incorrect indentation level 8, expected level should be 2\.